### PR TITLE
Track avro export costs [VS-769]

### PR DESCRIPTION
**Pull request by rsasch**
Use existing python function to track avro file generation cost.

Test run here: https://app.terra.bio/#workspaces/gvs-dev/RSA%20-%20GVS%20Quickstart%20V2%20/job_history/8890e4c5-8521-4555-96a1-c96dd54b2fd8.

### CODE REVIEW

**Review by RoriCremer — APPROVED** [review]

File: scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl [hunk 1]
```diff
@@ -223,7 +225,7 @@ task ExtractFromSuperpartitionedTables {
     }
 
     runtime {
-        docker: "gcr.io/google.com/cloudsdktool/cloud-sdk:409.0.0-alpine"
+        docker: "us.gcr.io/broad-dsde-methods/variantstore:2023-03-07-alpine-v2"
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:228]: this one is v2 and the others are not, is this intentional?

File: scripts/variantstore/wdl/extract/run_avro_query.py [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(allow_abbrev=False, description='Extract avro files from GVS BigQuery dataset')
+    parser.add_argument('--call_set_identifier', type=str,
+                        help='callset identifier used to track costs in cost_observability table', default='false')
+    parser.add_argument('--dataset_name',type=str, help='BigQuery dataset name', required=True)
+    parser.add_argument('--table_name',type=str, help='BigQuery table name', required=True)
+    parser.add_argument('--project_id', type=str, help='Google project for the GVS dataset', required=True)
+    parser.add_argument('--sql', type=str, help='SQL to run to extract AVRO data', required=True)
```
Comment (mcovarr) [inline scripts/variantstore/wdl/extract/run_avro_query.py:28]: nit, just '[Avro](https://avro.apache.org/docs/)'
```suggestion
    parser.add_argument('--sql', type=str, help='SQL to run to extract Avro data', required=True)
```

**Review by mcovarr — APPROVED** [review]
nice!

File: scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl (hunk 1 above)
Comment (rsasch) [reply scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl:228]: thanks — made them consistent

**Pull request merged**